tests/__fixtures__/cardanoGetNativeScriptHash.js

```diff
@@ -0,0 +1,381 @@
+import { CardanoNativeScriptHashDisplayFormat, CardanoNativeScriptType } from '../../src/js';
+
+export default {
+    method: 'cardanoGetNativeScriptHash',
+    setup: {
+        mnemonic: 'mnemonic_all',
+    },
+    tests: [
+        {
+            description: 'PUB_KEY script',
+            params: {
+                script: {
+                    type: CardanoNativeScriptType.PUB_KEY,
+                    keyHash: 'c4b9265645fde9536c0795adbcc5291767a0c61fd62448341d7e0386',
+                },
+                displayFormat: CardanoNativeScriptHashDisplayFormat.HIDE,
+            },
+            result: {
+                scriptHash: '29fb5fd4aa8cadd6705acc8263cee0fc62edca5ac38db593fec2f9fd',
+            },
+            // TODO GK MS - fix versions
```

Comment:
Just a heads up that this is here

Reply:
Yes, I think these can be removed as it seems the FW version will indeed be 2.4.3.